doc/Makefile

```diff
@@ -48,6 +48,16 @@ html-noplot:
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html/stable."
 
+html-pattern:
```

Comment:
I'm not sure we need a new target here... Is it hard or problematic to just use the html target?

Reply:
I could not find a way to do it cleanly in the html target, suggestions more than welcome! The main reason: what should the default pattern be? If you set the default pattern in the Makefile, then it will override any change in `conf.py`: the pattern is set on the command-line of the `sphinx-build` command. I would expect `conf.py` to have the last word.

Reply:
I haven't tried, but can't you do something like
```
ifdef EXAMPLE_RE
    EXAMPLE_OPT := -d blah=$(EXAMPLE_RE)
```

Reply:
Outside of any target, that is